lighthouse-core/test/audits/image-size-responsive-test.js

```diff
@@ -23,7 +23,8 @@ function generateImage(clientSize, naturalSize, props, src = 'https://google.com
       right: clientSize.displayedWidth,
     },
   };
-  Object.assign(image, clientSize, naturalSize, clientRect, props);
+  // eslint-disable-next-line max-len
```

Comment:
this one seems fine to let flow to multiple lines or do your object spread update :)